website/docs/ref/vite-plugin.md

```diff
@@ -0,0 +1,46 @@
+# Vite Plugin
+
+It's a good practice to use compiled message catalogs during development.
+However, running [`compile`](/docs/ref/cli.md#compile) everytime messages are changed soon becomes tedious.
```

Comment:
Please make it a single-line statement (it's a common style for all the documentation)